cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -34,16 +34,13 @@ func bedMaxWig(infile string, database string, chromsizeFile string, outfile str
 	bed.Write(outfile, outlist, 7)
 }
 
-func WigChromToSlice(w []*wig.Wig, size int, chrom string) []float64 {
+func WigChromToSlice(w []wig.Wig, size int, chrom string) []float64 {
 	output := make([]float64, size)
 	for _, v := range w {
 		if v.Chrom == chrom {
-			for i := 0; i < len(output); i++ {
-				output[i] = v.Values[i].Value
-			}
+			output = v.Values
```

Comment:
Yes, current set up is to return the moment a wig chrom name matches the input chrom name in the third argument. It does not continue to search for more once it finds a match. I think this hasnt been a problem because use thus far has been with wigs with only one struct per chromosome.

Reply:
Don't need to fix this now, but you can you make a github issue about this so that we don't forget?